Jenkinsfile

```diff
@@ -724,13 +725,52 @@ stage('Build packages') {
 }
 */
 
+def deploy_docs() {
+  // Note: This code must stay in the Jenkinsfile to ensure that it runs
+  // from a trusted context only
+  sh(
+    script: '''
+      set -eux
+      rm -rf tvm-site
+      git clone -b $DOCS_DEPLOY_BRANCH --depth=1 https://github.com/apache/tvm-site
+      cd tvm-site
+      git status
+      git checkout -B $DOCS_DEPLOY_BRANCH
+
+      rm -rf tvm-site/docs
+      mkdir -p tvm-site/docs
+      tar xf ../docs.tgz -C tvm-site/docs
```

Comment:
wondering if this stuff should live in a script in tvm-site repo instead?

Reply:
see the comment above, it's explicitly not in a script so it guarantees that it executes from a trusted env (we could do something like the answers here https://stackoverflow.com/questions/56294299/how-do-you-specify-additional-trusted-files-with-the-jenkins-github-branch-sourc but I want to get this in first and maybe leave that for a clean up PR)

Reply:
discussed--we'll move the tvm-site specific stuff to that repo in a follow-on.